radix-engine-tests/tests/component.rs

```diff
@@ -113,3 +113,169 @@ fn blueprint_name_can_be_obtained_from_a_method() {
     // Assert
     assert_eq!(blueprint_name, "ComponentTest")
 }
+
+#[test]
+fn pass_bucket_to_other_component() {
```

Comment:
I think we can remove the pass_bucket and pass_proof tests as we have plenty of those. The pass bucket and proof though is the one we really need. Could we also add a pass vault and proof

Reply:
I am wondering how can I pass `Vault` returned by 1st component function using `ManifestBuilder` to 2nd component? I don't see any function like for `Bucket` or `Proofs` to take it and lookup by name. The only way to pass `Vault` between components is by cross-blueprint calls?

Reply:
It may not be possible in the manifest but should be possible to do in a function which the manifest calls